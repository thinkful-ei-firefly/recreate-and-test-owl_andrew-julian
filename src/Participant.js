import React from 'react';

export default function Participant(props) {
    let classes1;
    let classes2;
    let status;
    if (props.inSession === true) {
        classes1 = 'participant inSession'
    }
    else {
        classes1 = 'participant notInSession'
    }

    if (props.onStage === true) {
        classes2 = 'onStageStatus onStage';
        status = 'on stage';
    }
    else {
        classes2 = 'onStageStatus notOnStage';
        status = 'in session'
    }


    return (
        <div className={classes1} id={props.id}>
            <img src={props.avatar} alt="avatar"></img>
            <h4>{props.name}</h4>
            <div className={classes2}>
                <h6>&bull;</h6>
                <h5>{status}</h5>
            </div>
        </div>
    )
}