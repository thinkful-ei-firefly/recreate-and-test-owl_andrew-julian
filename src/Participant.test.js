import React from 'react';
import ReactDOM from 'react-dom';
import Participant from './Participant';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Participant />, div);
  ReactDOM.unmountComponentAtNode(div);
});
