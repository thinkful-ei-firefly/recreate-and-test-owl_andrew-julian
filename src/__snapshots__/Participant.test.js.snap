// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="participant notInSession"
>
  <img
    alt="avatar"
  />
  <h4 />
  <div
    className="onStageStatus notOnStage"
  >
    <h6>
      â€¢
    </h6>
    <h5>
      in session
    </h5>
  </div>
</div>
`;
